di-ipv-cimit-stub/lambdas/get-contra-indicators/src/main/java/uk/gov/di/ipv/core/getcontraindicators/GetContraIndicatorsHandler.java

```diff
@@ -6,36 +6,69 @@
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.message.StringMapMessage;
+import uk.gov.di.ipv.core.getcontraindicators.domain.ContraIndicatorItem;
 import uk.gov.di.ipv.core.getcontraindicators.domain.GetCiRequest;
 import uk.gov.di.ipv.core.getcontraindicators.domain.GetCiResponse;
+import uk.gov.di.ipv.core.library.persistence.items.CimitStubItem;
+import uk.gov.di.ipv.core.library.service.CimitStubItemService;
+import uk.gov.di.ipv.core.library.service.ConfigService;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
-import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
 
 public class GetContraIndicatorsHandler implements RequestStreamHandler {
 
     private static final Logger LOGGER = LogManager.getLogger();
     private static final ObjectMapper mapper = new ObjectMapper();
 
+    private final ConfigService configService;
+    private final CimitStubItemService cimitStubItemService;
+
+    public GetContraIndicatorsHandler() {
+        this.configService = new ConfigService();
+        this.cimitStubItemService = new CimitStubItemService(configService);
+    }
+
+    public GetContraIndicatorsHandler(
+            ConfigService configService, CimitStubItemService cimitStubItemService) {
+        this.configService = configService;
+        this.cimitStubItemService = cimitStubItemService;
+    }
+
     @Override
     public void handleRequest(InputStream input, OutputStream output, Context context)
             throws IOException {
         LOGGER.info(new StringMapMessage().with("Function invoked:", "GetContraIndicators"));
 
+        GetCiRequest getCiRequest;
         GetCiResponse response;
         try {
-            mapper.readValue(input, GetCiRequest.class);
+            getCiRequest = mapper.readValue(input, GetCiRequest.class);
         } catch (Exception ex) {
             LOGGER.error(
                     new StringMapMessage().with("Unable to parse input request", ex.getMessage()));
             throw ex;
         }
 
         // Initially returning empty CI's
-        response = new GetCiResponse(Collections.emptyList());
+        response = new GetCiResponse(getContraIndicators(getCiRequest.getUserId()));
 
         mapper.writeValue(output, response);
     }
+
+    private List<ContraIndicatorItem> getContraIndicators(String userId) {
+        List<CimitStubItem> cimitStubItems = cimitStubItemService.getCIsForUserId(userId);
```

Comment:
I think we can check to cimitCtubItems's null value and then return an empty list.

Reply:
Don't think it will return null, can return an empty list of CimitStubItem. Added a test for that.